plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -169,6 +169,23 @@ func findNodesThatFit(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.No
 			if err != nil {
 				return api.NodeList{}, FailedPredicateMap{}, err
 			}
+			//add the extender failed info to failedPredicateMap
+			for _, filteredNode := range filtered {
```

Comment:
Why you prefer checking outside instead of that Filter() tells you why it failed some nodes?